src/MSAL.PCL/Internal/Cache/TokenCacheAccessor.cs

```diff
@@ -43,7 +43,7 @@ public void SaveRefreshToken(RefreshTokenCacheItem refreshTokenItem)
             TokenCachePlugin.SaveRefreshToken(refreshTokenItem);
         }
 
-        public IList<AccessTokenCacheItem> GetTokens(TokenCacheKey tokenCacheKey)
+        public IList<AccessTokenCacheItem> GetAccessToken(TokenCacheKey tokenCacheKey)
```

Comment:
should `TokenCachePlugin.SaveToken` change to `TokenCachePlugin.SaveAccessToken` as well? :)